boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/model/MapSearchResultDTO.kt

```diff
@@ -0,0 +1,20 @@
+package base.boudicca.publisher.event.html.model
+
+
+data class MapSearchResultDTO(
```

Comment:
while we use this for a map, would `LocationSearchResult` and `/locationSearch` not be better namings for the DTO and endpoint?

Reply:
hm... good point, but i think `locationSearch` is a bit misleading because you don't actually search for locations, but the result is grouped by locations... but yeah, MapSearch is also not a great name... will think a bit about that one :D

Reply:
hm... i think i would leave it as `mapSearch` for now. the reason is that this is only an internal and not a public api (this is in the html-publisher) so it will only ever be used for the map and can be quite specific to that